src/utils/paginatablePageGenerator.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env node */
```

Comment:
can we keep utils split by their meaning - it was in the blog - we should have blog component, paginator and utils there instead of very long names ... and it's def not a global util

Reply:
There are few reasons why I moved it here:
- It uses on global level (on page generation process in the `gatsby-node.js`) and not in the Paginator itself
- By putting it in the `utils` folder override rules for eslint applied automatically 
https://github.com/iterative/dvc.org/blob/233e9334a5b1c1a0a375e77d8eba8302ec18c465/.eslintrc.json#L36-L42
I can rename it to something else, but I prefer clarity over shorteness

Reply:
* linters should never dictate code structure, so I would not consider a good argument
* global usage is a good point - we had a debate when implementing this and at that point decided to keep under some kind of `blog/utils` since it's related to blog ... it's totally fine for gatsby-node to use utils/code from different levels

Reply:
Totally agree with first point. It's just conveniently for this case